examples/conf-quickstart/broker/runtime.properties

```diff
@@ -0,0 +1,16 @@
+druid.service=druid/broker
+druid.port=8082
+
+# HTTP server threads
+druid.broker.http.numConnections=5
+druid.server.http.numThreads=8
+
+# Processing threads and buffers
+druid.processing.buffer.sizeBytes=256000000
+druid.processing.numThreads=2
+
+# Query cache (we use a small local cache)
+druid.broker.cache.useCache=true
+druid.broker.cache.populateCache=true
+druid.cache.type=local
+druid.cache.sizeInBytes=10000000
```

Comment:
@himanshug Caching on broker can be faster for smaller queries. However, with larger queries the broker can also become a bottleneck. So it depends a lot on the the types of queries being run.

In practice someone will have to benchmark and see what the trade-offs are. It is also possible to enable on both historical and broker and adjust cache settings to avoid bottlenecks on the brokers